lib/Classifiers/Classifier.php

```diff
@@ -97,8 +107,12 @@ public function classifyFiles(string $model, array $queueFiles, int $timeout): \
 					}
 					// Check file dimensions
 					$dimensions = @getimagesize($path);
-					if (isset($dimensions) && $dimensions !== false && ($dimensions[0] > 1024 || $dimensions[1] > 1024)) {
-						$this->logger->debug('File dimensions are too large for classifier: ' . $files[0]->getPath());
+					if (isset($dimensions) && $dimensions !== false &&  ($dimensions[0] != self::TEMP_FILE_DIMENSION && $dimensions[1] != self::TEMP_FILE_DIMENSION)) {
+						$this->logger->debug('dimension of ' . $queueFile->getFileId() . ' does not contain expected dimension ' . self::TEMP_FILE_DIMENSION . ' dimensions ' . $dimensions[0] . '/' . $dimensions[1]  );
+					}
```

Comment:
Can you remove this change, as I think it would fail if a file is not square.

Reply:
Done,

it was only logging for now, but if we are being honest with ourself here. It will take a fair bit of time for getting feedback on the strange behvor of nextlcoud server.